### ISSUE TITLE
GPU v2(angel)Disck Shaders Cache and GPU Asynchronous SMO 

### ISSUE THREAD

**Issue by emulationchannel** [Ryujinx/Ryujinx#855]
SMO GPU v2 # 789 requires Disk Shaders Cache to reduce choking on emulation part Asynchronous GPU requirement for better speed performance

**Comment by ReaperOfSouls1909** [comment]
ryujinx has Async Gpu already but feel free to add a disk cache

**Comment by emmauss** [comment]
Closing this because
1. this is not an issue
2. it doesn't concern the main branch
3. the branch it relates to isn't even merged, and is, though ready, not complete in a sense.
4. If you think it's easy to "fix/improve" issues just by giving suggestions, obvious ones too, please feel free to add one.
5. The issue tracker is not for suggestions. Those should go to the discord server.

TLDR: just closing it
